### ISSUE TITLE
Loss NAN

### ISSUE THREAD

**Issue by jqyang22** [jonathanventura/urban-tree-detection#5]
Hi Jonathan,

Thanks for sharing the great code!

I encountered a situation where the loss is all NAN while running the code without changing the model, and even reducing the learning rate didn't solve the problem. Can you please help me with this if possible?

Thanks a lot.

**Comment by jonathanventura** [comment]
Hi, what data are you training on?

**Comment by jqyang22** [comment]
Hi, thank you for your prompt reply! I think I use the default setting and the shared data used in this code. I just change the local file path. For example, the "path to dataset" for prepare.py is the folder "/urban-tree-detection-data" containing all the files in the link "https://github.com/jonathanventura/urban-tree-detection-data/".
